Add page for single-cell specific resources

allyhawkins opened the pull request:
Closes #18 

Here I added an additional page for single-cell specific resources. I modeled this after the `Additional Resources` page from the RRP site since this was linked in the issue. I chose to include links to relevant tutorials and vignettes as well as any relevant papers for some common topics for single-cell that might be of interest. 

A few questions: 

- Are we okay with this style of listing resources? Do we want to add anymore context here? 
- Are there things I'm missing or should not include? 
- We still have a separate page for consultation sessions that includes different resources and I chose to keep this separate. I think that's what we want since that page has things like scpca on it whereas I see this being links to guides/ tutorials/ papers to help complement our instruction. 

The only section I haven't fully filled out yet is on differential abundance, because I need to dig a little deeper to decide what I want to put there. But I wanted to file this to get some initial thoughts and answers to these questions to at least make sure formatting was how we wanted it.

jashapiro reviewed (commented):
Overall I think this looks good, but I'd update citations to remove journal names... Maybe add authors and years instead.  Because I am anti-journal like that.

A couple cell typing links that might be worth linking:
One is the 10X page about cell typing: https://www.10xgenomics.com/resources/analysis-
guides/web-resources-for-cell-type-annotation

Then maybe a review paper? http://dx.doi.org/10.1093/bfgp/elaa003 and/or https://doi.org/10.1016/j.csbj.2021.10.027
(I'm a bit less enamored of linking to these papers, but see what you think)

sjspielman commented on workshop/single-cell-resources.md:
```diff
@@ -0,0 +1,82 @@
+It includes multiple types of resources for various topics in single-cell RNA-seq analysis, but does not represent the complete breadth of analysis topics.
+Resources are listed by topic and in alphabetical order, not in order of recommendation.
+
+**Table of contents**
```
Do we want to keep the doctoc comments here for future updates, as in - https://github.com/AlexsLemonade/training-specific-template/issues/98

sjspielman replied:
Still wondering about this comment..?

allyhawkins replied:
Ah sorry I meant to address this, I'll add in doctoc now.

sjspielman commented on workshop/single-cell-resources.md:
```diff
@@ -0,0 +1,82 @@
+
+## CITE-seq
+
+- [OSCA chapter on integration with protein abundance](http://bioconductor.org/books/3.16/OSCA.advanced/integrating-with-protein-abundance.html)
```
```suggestion
- [OSCA chapter on integrating with protein abundance](http://bioconductor.org/books/3.16/OSCA.advanced/integrating-with-protein-abundance.html)
```

allyhawkins commented:
Okay I updated the references to remove the journals and include the authors and year. I formatted them like we do with in text citations, but I can change that if others have citation opinions. 

I also added a few more references, some Josh mentioned, another one for DE that I like, and also some for differential abundance. I tried to find a good tutorial or vignette on DA, but the best one is the OSCA one and even then... meh. So I mostly included papers - one comparison of methods paper on biorxiv, and then two methods papers - one of which is from the scvi group that I had looked into before that I like. This is ready for another round of review.

jashapiro reviewed (approved):
LGTM! I had a thought about going through and changing a bunch of en-dashes to em-dashes, but I will refrain.

jashapiro commented on workshop/single-cell-resources.md:
```diff
@@ -8,7 +8,9 @@ Please note, this is not an exhaustive list.
-**Table of contents**
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
```
```suggestion
**Table of Contents** 
```

sjspielman replied:
Just peeked at this, and we can do - https://github.com/thlorenz/doctoc#specifying-a-custom-toc-title

tested locally, and it needs the `**` - 
```
doctoc blah.md --title **Table of Contents**
```

allyhawkins replied:
Thanks Stephanie! Should be fixed now using that trick.

jashapiro replied:
Once it is removed, it stays removed, so either way would have been fine!
